web/src/components/NavigationBarScrollContainer.tsx

```diff
@@ -53,22 +53,22 @@ const ScrollContainer = styled.div<{ showArrowContainer: boolean }>`
   }
 `
 
-const getInitialScrollPosition = (activeIndex: number): number => {
+const getActiveItemScrollPosition = (activeIndex: number): number => {
   const navigationBar = document.getElementById('navigation-bar')
   if (!navigationBar || activeIndex === 0) {
     return 0
   }
   const navigationBarOffset = navigationBar.offsetLeft
-  const elementOffset = navigationBar.getElementsByTagName('div')[activeIndex]?.offsetLeft
-  return elementOffset ? navigationBarOffset + elementOffset : 0
+  const navigationItem = navigationBar.getElementsByClassName('navigation-item')[activeIndex] as HTMLElement | null
+  return navigationItem?.offsetLeft ? navigationItem.offsetLeft - navigationBarOffset : 0
```

Comment:
Optional:
``` ts
  const navigationBar = document.getElementById('navigation-bar')
  const navigationItem = navigationBar?.getElementsByClassName('navigation-item')[activeIndex]
  if (!navigationBar || !(navigationItem instanceof HTMLElement)) {
    return 0
  }
  return navigationItem.offsetLeft - navigationBar.offsetLeft
```

Reply:
At least the check for `activeIndex === 0` is not needed anymore and should be removed thanks to your adjustments in calculating the scroll position.

Reply:
fixed

Reply:
Check for `navigationItem?.offsetLeft` is not necessary. Not sure why our linitng does not mark that though